src/csw/cswClientWrapper.ts

```diff
@@ -148,6 +148,20 @@ export class CswClientWrapper {
                 default:
                   return val;
               }
+            case isBoolean(key): {
+              // Python booleans notations are capitalized, we should parse it to JS notation.
+              if (typeof val === 'string') {
+                const isBooleanValue = ['True', 'False'].includes(val);
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                const pythonBooleanStringMap: Record<string, boolean> = { True: true, False: false };
```

Comment:
python?

Reply:
Yeah, this is cswClientWrapper, this function is used to transform the incoming PYCSW data and booleans in python are written as True and False in opposed to JS, the values also received as a string. this case should handle it

Reply:
Think it's mostly DB(SQL), rather then python

Reply:
basically the var might be called "kuku" but .... just remove **python** preffix